docset/winserver2022-ps/appx/Add-AppxPackage.md

```diff
@@ -72,54 +73,102 @@ The **Add-AppxPackage** cmdlet adds a signed app package to a user account.
 ```
 
 This command will register an update to an existing app, but will not do so until the next launch of the app.
 
 ### Example 3: Add a disabled app package in development mode
-```
-PS C:\> $ManifestPath = (Get-AppxPackage -Name "*WindowsCalculator*").InstallLocation + "\Appxmanifest.xml"
-PS C:\> Add-AppxPackage -Path $ManifestPath -Register -DisableDevelopmentMode
+```powershell
+$ManifestPath = (Get-AppxPackage -Name "*WindowsCalculator*").InstallLocation + "\Appxmanifest.xml"
+Add-AppxPackage -Path $ManifestPath -Register -DisableDevelopmentMode
 ```
 
 This command gets the full path of the package manifest file of an installed Windows Store app, and then registers that package.
 You can use *DisableDevelopmentMode* to register an application that is staged by the **StagePackageAsync** API, has been disabled, or has become corrupted during testing.
 
 ### Example 4: Add an app along with its optional packages
-```
-PS C:\> Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msixbundle" -ExternalPackages "C:\Users\user1\Desktop\optionalpackage1.msix","C:\Users\user1\Desktop\optionalpackage2.msixbundle"
+```powershell
+Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msixbundle" -ExternalPackages "C:\Users\user1\Desktop\optionalpackage1.msix","C:\Users\user1\Desktop\optionalpackage2.msixbundle"
 
-PS C:\> Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msixbundle" -OptionalPackages "29270sandstorm.OptionalPackage1_gah1vdar1nn7a"
+Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msixbundle" -OptionalPackages "29270sandstorm.OptionalPackage1_gah1vdar1nn7a"
 ```
 
 This command adds an app package along with its optional packages. It is an atomic operation which means that if the app or its optional packages fail to install, the deployment operation will be aborted
 
 ### Example 5: Install only the required section of a streaming app
-```
-PS C:\> Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msixbundle" -RequiredContentGroupOnly
+```powershell
+Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msixbundle" -RequiredContentGroupOnly
 ```
 
 This command adds an app package but only installs the required section of a streaming app. Calling this command again without the RequiredContentGroupOnly flag proceeds to install the rest of the application in the order defined by the AppxContentGroupMap.xml
 
 ## PARAMETERS
 
+### -AllowUnsigned
+
+Allows adding an unsigned package.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: AddSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AppInstallerFile
+Runs an appinstaller file and allows the user to install all of the defined packages with a single click.
+For more information, see [Create an App Installer file manually](https://github.com/microsoftdocs/msix-docs/blob/master/msix-src/app-installer/how-to-create-appinstaller-file.md).
```

Comment:
Shouldn't this link be `/windows/msix/app-installer/how-to-create-appinstaller-file` ?

Reply:
@get-itips - nice catch. Agree.

Reply:
Thanks for the feedback. I made the link relative.